### PR TITLE
Adds html pages and content based on wireframe sketches

### PR DESCRIPTION
**Pull request by krichmond06**

### CODE REVIEW

File: contact.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="Description" content="INF 7420 Project">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Document</title>
```
Comment (axamei) [inline contact.html]: I recommend changing the title of your pages.

File: contact.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      </li>
+    </ul>
+  </nav>
+  <h1>Get in Touch</h1>
+    <h2>Contact Me</h2>
+       <p>I'll insert a contact form here after we learn about web forms in Week 9.</p>
+    <img src="images/selfpohtoflyfish.jpg" alt="photo of katie richmond dressed in fly fishing gear and standing on a paved path with trees in background"/>
+       <p>Adventures abound wherever you are!</p>
+       <p>Thanks for visiting my page!</p>
+  <div id="footer">
```
Comment (axamei) [inline contact.html]: Instead of the div, I recommend using the `<footer>` element.

**Review by axamei — CHANGES_REQUESTED** [review]
In addition to the footer, I recommend adding a `<header>` and `<main>` element with the appropriate elements nested inside those tags. This will create a more meaningful structure for your pages.

**Comment by krichmond06** [comment]
Thank you for your suggestions, Axa! I have made the suggested changes and pushed the new commit. Thanks!

**Review by axamei — CHANGES_REQUESTED** [review]
I recommend you change the title of your pages / the text between the `<title></title>` tags in the head of the document.

**Comment by krichmond06** [comment]
Oops, I changed the title on the index page, but forgot to change it on the others. I have changed it now, but am running into an issue with pushing the changes. I keep getting this message when trying to add files to commit: "Another git process seems to be running in this repository, e.g.
an editor opened by 'git commit'. Please make sure all processes
are terminated then try again. If it still fails, a git process
may have crashed in this repository earlier:
remove the file manually to continue." I've closed all other processes, so I'm not sure what the issue is?

**Comment by krichmond06** [comment]
I believe I deleted the correct file that was causing the issue and have pushed the changes now.

**Review by axamei — APPROVED** [review]

**Pull request merged**